lib/smart_answer_test_helper.rb

```diff
@@ -11,10 +11,24 @@ def self.responses_and_expected_results
     Dir[data_path.join('*-responses-and-expected-results.yml')]
   end
 
+  def self.configurations
+    yaml = File.read(data_path.join('configurations.yml'))
+    YAML.load(yaml)
+  end
+
+  def self.default_configuration
+    configurations.fetch('default')
+  end
+
```

Comment:
Is there any reason for making `SmartAnswerTestHelper.configurations` and `SmartAnswerTestHelper.default_configuration` public?

It seems that `SmartAnswerTestHelper#configuration` will be the method that will be exposed to the regression tests. What about adding more context about how it is going to be used in the commit notes?